lib/label.ml

```diff
@@ -29,12 +29,12 @@ module Relation = struct
     | Le -> Fmt.string ppf "<="
 
   let compare = function
-    | Eq -> ( = )
-    | Neq -> ( <> )
-    | Lt -> ( < )
-    | Le -> ( <= )
-    | Gt -> ( > )
-    | Ge -> ( >= )
+    | Eq -> ( =)
+    | Neq -> ( <>)
+    | Lt -> ( <)
+    | Le -> ( <=)
+    | Gt -> ( >)
+    | Ge -> ( >=)
```

Comment:
regressions here

Reply:
fixed with the last ocamlformat build (https://github.com/ocaml-ppx/ocamlformat/commit/81279cda0be6882fb8d821eea1bb472f0941b307)